pages/apis/rest_api/clusters.md

```diff
@@ -227,3 +236,116 @@ Error responses:
   <tr><th><code>422 Unprocessable Entity</code></th><td><code>{ "message": "Reason the cluster couldn't be deleted" }</code></td></tr>
 </tbody>
 </table>
+
+## Cluster queues
+
+[Cluster queues](https://buildkite.com/docs/agent/clusters#set-up-a-cluster-set-up-queues) are discrete groups of agents within a cluster. Pipelines in that cluster can target cluster queues to run jobs on agents assigned to those queues.
+
+
+### Cluster queue data model
+
+<table class="responsive-table">
+  <tbody>
+    <tr><th><code>id</code></th><td>ID of the queue</td></tr>
+    <tr><th><code>graphql_id</code></th><td><a href="/docs/apis/graphql-api#graphql-ids">GraphQL ID</a> of the queue</td></tr>
+    <tr><th><code>key</code></th><td>The queue key</td></tr>
+    <tr><th><code>description</code></th><td>Description of the queue</td></tr>
+    <tr><th><code>url</code></th><td>Canonical API URL of the queue</td></tr>
+    <tr><th><code>web_url</code></th><td>URL of the queue on Buildkite</td></tr>
+    <tr><th><code>cluster_url</code></th><td>API URL of the cluster the queue belongs to</td></tr>
+    <tr><th><code>dispatch_paused</code></th><td>Indicates whether the queue is paused</td></tr>
```

Comment:
Since "dispatching" jobs is a bit of a new concept coming in with these changes, I think it would be good to use that word here. Maybe:

```suggestion
    <tr><th><code>dispatch_paused</code></th><td>Indicates whether the queue is paused from dispatching jobs to agents</td></tr>
```

What do you think?

Reply:
@mbelton-buildkite something about "paused from" that I'm not sure about... what do you think of this: 

"Indicates whether job dispatch to associated agents is paused for this queue"

Reply:
That could work. The only hesitations I have with it are "job dispatch" being a thing rather than an action and moving "queue" so far back in the description. I think it makes it sound a bit more complicated than it is. Another option could be:

```suggestion
    <tr><th><code>dispatch_paused</code></th><td>Indicates whether the queue has paused dispatching jobs to agents</td></tr>
```

I'm happy for you to decide which one to use. 😊

Reply:
Thanks @mbelton-buildkite ! How about:
"Indicates whether the queue has paused dispatching jobs to associated agents" ? Im keen to put 'associated' in there so it doesnt read as though _all_ agents are affected